test/users/engin/retInnerFunction.future

```diff
@@ -0,0 +1,5 @@
+error message: returning an inner function that uses outer-local vars
```

Comment:
Not a huge deal, but I'd call this a bug (I think any internal error is a bug).  I reserve "error message" futures to be cases where we generate a user-facing error message but it's not as good or helpful as it could be and we'd like to improve it (and don't consider internal errors to be in that same class of user-facing error messages I suppose).